src/index.js

```diff
@@ -0,0 +1,3 @@
+import _ from 'lodash';
```

Comment:
**You are missing the last step of your requirements:**
![Screen Shot 2021-08-10 at 20 35 49](https://user-images.githubusercontent.com/22258274/128956381-5c0597fe-4cbf-44d0-845d-3a2e215fd167.png)

**Kindly work with using ES6 modules to write modular JavaScript with proper ES6 syntax.**
- use of arrow functions instead of the function keyword
- use of import and export instead of require or module.exports
- proper usage of const and let keywords